train/train_process.py

```diff
@@ -75,13 +73,14 @@ def run_training_loop(self):
             model_id = net_utils.model_id(
                 self.game.load_model(model_path, self.net_type))
             training_games_dir = os.path.join(
-                self.cfg["games_dir"], f"{run_id}_{iter_num}_{model_id}")
+                self.cfg["games_dir"], "{0}_{1:05d}_{2}".format(run_id, iter_num, model_id))
 
             self._self_play(model_path, training_games_dir)
             new_model = self._train(model_path, training_games_dir)
 
             new_model_path = self._save_model(new_model)
-            compare_models(model_path, new_model_path)
+            _w1, _w2, _d = self._compare_models(model_path, new_model_path)
+            # TODO do something with comparison result

```

Comment:
Let's add something simple for now - 
if _w2 <= _w1, log a warning and don't switch models (continue to the next playing round). 
This will also show us if we have reached rounds with all-ties

Reply:
added a switching_threshold and warning_threshold